bot/cogs/defcon.py

```diff
@@ -223,17 +197,13 @@ def mod_log(self) -> ModLog:
         self.mod_log.ignore(Event.guild_channel_update, Channels.defcon)
         await self.channel.edit(topic=new_topic)
 
-    def build_defcon_msg(self, change: str, e: Exception = None) -> str:
-        """
-        Build in-channel response string for DEFCON action.
-
-        `change` string may be one of the following: ('enabled', 'disabled', 'updated')
-        """
-        if change.lower() == "enabled":
+    def build_defcon_msg(self, action: Action, e: Exception = None) -> str:
+        """Build in-channel response string for DEFCON action."""
+        if action is Action.ENABLED:
             msg = f"{Emojis.defcon_enabled} DEFCON enabled.\n\n"
-        elif change.lower() == "disabled":
+        elif action is Action.DISABLED:
             msg = f"{Emojis.defcon_disabled} DEFCON disabled.\n\n"
-        elif change.lower() == "updated":
+        elif action is Action.UPDATED:
             msg = (
                 f"{Emojis.defcon_updated} DEFCON days updated; accounts must be {self.days} "
```

Comment:
Fixes a small issue with the formatting. It was displaying the hours, minutes, and seconds as well.
```suggestion
                 f"{Emojis.defcon_updated} DEFCON days updated; accounts must be {self.days.days} "
```

If you feel like it you could also make the message show singular "day" when it's 1 day.

Reply:
That's a nice spot. I'll also make it a QoL change now and make it `day` when it's 1 day.